clusterloader2/pkg/prometheus/manifests/0prometheus-operator-deployment.yaml

```diff
@@ -39,7 +42,27 @@ spec:
             memory: 100Mi
         securityContext:
           allowPrivilegeEscalation: false
-          readOnlyRootFilesystem: true
+      - args:
+        - --logtostderr
+        - --secure-listen-address=:8443
+        - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
+        - --upstream=http://127.0.0.1:8080/
+        image: quay.io/brancz/kube-rbac-proxy:v0.8.0
```

Comment:
I am not sure what this container is doing, but either way if we want to use it then we will need to mirror it to gcr (in the past we had issues, where our tests were failing, because quay.io was down.

Reply:
Just a note: There are also [talks in kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/issues/1785) to make automatic cloning of kube-rbac-proxy. Plus we (kube-rbac-proxy maintainers) want to donate that to k8s in the future when we find time to actually do it.

Reply:
it seems 0.8.0 is available as per https://github.com/kubernetes-sigs/kubebuilder/pull/1955, updating this PR accordingly.

Reply:
@marseel: I am new to this ecosystem, but can `gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` be referenced her as well or can you advise how I can help in having a `gcr.io/k8s-testimages/quay.io/brancz/kube-rbac-proxy:v0.8.0`?

tldr: kube-rbac-proxy is protects endpoints by executing a token and subject access review against k8s api to validate if service account token in flight has permissions to query metrics.